packages/tasks/src/model-libraries.ts

```diff
@@ -787,6 +787,13 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		filter: true,
 		countDownloads: `path_extension:"sentis"`,
 	},
+	"sana": {
```

Comment:
```suggestion
	"Sana": {
```

Since the tags in the repo are w/ capital `S`: https://huggingface.co/models?other=Sana